File: src/gestorAplicacion/Veterinaria/Inventario.java
package gestorAplicacion.Veterinaria;

import java.util.ArrayList;

public class Inventario {
    public static ArrayList<Medicamento> Medicamentos= new ArrayList<Medicamento>();
    	
	public static ArrayList<Medicamento> getMedicamentos() {
        return Medicamentos;
    }

    public static void setMedicamentos(Medicamento nuevoMedicamento) {
        Medicamentos.add(nuevoMedicamento);
    }
    
}

File: Python/gestorAplicacion/Veterinaria/Inventario.py
from gestorAplicacion.Veterinaria import Medicamento

class Inventario:
    
    Medicamentos=[]
    
    @classmethod
    def getMedicamentos(cls):
        return cls.Medicamentos
    
    @classmethod
    def AñadirMedicamento(cls, nuevo):
        cls.Medicamentos.append(nuevo)

    
    

